### DISCUSSION TITLE
Hide unreleased Movies or TV Shows from Trakt watchlist

### DISCUSSION THREAD

**Discussion by KalAvroniev** [umbrellaplug/umbrellaplug.github.io#329] [Q&A] [answered]
Does anyone know a way to hide unreleased Movies or TV Shows from Trakt watchlists?

I looked at all the settings and could not find anything. 
The reason I'm asking is that it gets annoying to have to navigate several pages to get to the playable items.

If not could be another enhancement suggestion.

**Comment by SJ1O** [comment]
I'm sure I done this. Go onto trakt website view your watchlist and filter out unreleased movies. Add-ons just pull from trakt I was led to believe so I don't think it's possible for add-ons to edit trakt lists. I could be wrong but I'm sure that's what I have read. What would be cool is if there was a way to only view digital releases :)

**Comment by adamosborne83** [comment]
> I'm sure I done this. Go onto trakt website view your watchlist and filter out unreleased movies. Add-ons just pull from trakt I was led to believe so I don't think it's possible for add-ons to edit trakt lists. I could be wrong but I'm sure that's what I have read. What would be cool is if there was a way to only view digital releases :)

Discover Movies- Released this month (Digital and DVD)

**Comment by adamosborne83** [comment, marked as answer]
> Does anyone know a way to hide unreleased Movies or TV Shows from Trakt watchlists?
> 
> I looked at all the settings and could not find anything. 
> The reason I'm asking is that it gets annoying to have to navigate several pages to get to the playable items.
> 
> If not could be another enhancement suggestion.

Settings-General- Hide movies in cinema and unreleased.

**Reply by KalAvroniev** [reply]
Thanks so much, works perfectly

**Comment by Omarscomin** [comment]
> Does anyone know a way to hide unreleased Movies or TV Shows from Trakt watchlists?
> 
> 
> 
> I looked at all the settings and could not find anything. 
> 
> The reason I'm asking is that it gets annoying to have to navigate several pages to get to the playable items.
> 
> 
> 
> If not could be another enhancement suggestion.

Stop adding unreleased content to your watchlists😇🤷🏻‍♀️😂
